src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+</head>
+<body>
+  <div class = "container">
+    <header>
+     <h1>Leaderboard</h1>
+    </header>
+    <main>
+     <section class = "scores-container">
+      <div class="title">
+       <h2>Recent scores</h2>
+        <button type="button">Refresh</button>
+      </div>
+      <div class="score-list-container" id="scoreListContainer" >
+       
+      </div>
+     </section>
+     <section class = "add-scores">
+      <h2>Add your scores</h2>
+      <form action="#" id="add-form">
+       <input type="text" name="name" id="p-name" placeholder="Your name">
+       <input type="number" name="score" id="scores" placeholder="Your name">
+      <button type="Submit" id="add-btn">Submit</button>
+      </form>
+
+     </section>
```

Comment:
- [ ] Good job so far project is very well implemented :+1: But please make your project match with the wireframe as required you can see some differences with your submit button position and also your displaying list border. Please fix it  cause as a developer is very important to pay attention :smile:  
Check the screenshots below :arrow_down: 

#### Your design
![Screenshot from 2022-08-30 15-59-24](https://user-images.githubusercontent.com/6615394/187485048-4cf1a2c5-0af7-4e17-a0ee-9ed162c5d79d.png)

#### Original wireframe
![Screenshot from 2022-08-30 15-59-18](https://user-images.githubusercontent.com/6615394/187485056-f7dca197-f346-4d06-b076-8002e156321c.png)